src/eko/matching_conditions/n3lo/f_functions.py

```diff
@@ -0,0 +1,6980 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=too-many-lines
+r"""
+This module contains some Mellin transformations for 5th order harmonics sum.
+Notation refers to:
+
+    - :cite:`Blumlein:2009ta`. Johannes Blumlein. Structural Relations of Harmonic Sums and Mellin Transforms up to Weight w = 5. Comput. Phys. Commun., 180:2218-2249, 2009. arXiv:0901.3106, doi:10.1016/j.cpc.2009.07.004.
+
+Mellin transform is defined with the convention x^(N).
+F19, F20, F21 are not present explicitly in the paper
+"""  # pylint: disable=line-too-long
+import numba as nb
+import numpy as np
+
+
+@nb.njit("c16(c16,c16,c16,c16)", cache=True)
+def F17(n, S1, S2, S3):
```

Comment:
I'd suggest to split each function in a separate module

Reply:
Mainly, yes - and to get rid of the 7k lines file, and better maintainability (imagine someone discover Fxy is wrong by tomorrow)

Reply:
Then you can make a subpackage:
```
f_functions/
  __init__.py
  f17.py
  ...
```

Reply:
okay this is fine